exporter/elasticsearchexporter/factory.go

```diff
@@ -129,6 +143,11 @@ func createMetricsExporter(
 	cf := cfg.(*Config)
 	logConfigDeprecationWarnings(cf, set.Logger)
 
+	// Workaround to avoid rejections from Elasticsearch
+	// TSDB does not accept 2 documents with the same timestamp and dimensions
```

Comment:
Could you expand on this a little please? How does setting MaxSizeItems to zero help? I've taken a stab at it.

```suggestion
	// Workaround to avoid rejections from Elasticsearch.
	// TSDB does not accept 2 documents with the same timestamp and dimensions.
	//
	// Setting MaxSizeItems ensures that the batcher will not split a set of
	// metrics into multiple batches, potentially sending two metric data points
	// with the same timestamp and dimensions as separate documents.
```